brainspell/test_tornado.py

```diff
@@ -131,28 +134,51 @@ def test_procfile():
         assert filename in os.listdir()
 
 
-""" TODO: get selenium testing working
-def test_existence(): #Using selenium testing
+
+def test_existence(): #Using selenium testing to verify existence of site elements
     driver.get("https://brainspell.herokuapp.com")
```

Comment:
Hey, this shouldn't be calling the Heroku server; it should be testing the version of Brainspell running locally